docs/content/utilities/typography.md

```diff
@@ -103,6 +102,22 @@ Change the font weight, styles, and alignment with these utilities.
 <p class="user-select-none">User Select None</p>
 ```
 
+## Word-break
+There are two utilities for adjusting how lines and words of text break when they exceed the width of their containing element:
+
+1. `break-word` sets `word-break: break-word` and `overflow-wrap: break-word`, which will only break words if they would exceed the line length _after wrapping_.
+
+2. `wb-break-all` sets `word-break: break-all`, will force a word to break regardless of whether it's shorter than the line length, per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#Values):
+
+    > **Note:** In contrast to `word-break: break-word` and `overflow-wrap: break-word` (see [`overflow-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap)), `word-break: break-all` will create a break at the exact place where text would otherwise overflow its container (even if putting an entire word on its own line would negate the need for a break).
```

Comment:
These look great. I think the docs are self-explanatory, I don't think you need to display the _Note_ section (but maybe keep the link?). I don't feel that strongly though, so if you want to keep them in here that's fine.